assignments/types.json

```diff
@@ -1,3 +1,6 @@
 {
-    "4": "Liberation"
```

Comment:
what happened to type `4`?

Reply:
This was a misinformation campaign by the Automatons to disrupt our API tracking 

I was mistaken. originally type.json was for assignment['setting']['type'], which seems to be 4 regardless of the assignment, so far at least. So it was just a blunder on my part. - assignment/types.json has been moved to tasks/task/type.json to better reflect where and what the file was supposed to be representing.